Fix sidebar icons alignment

Usiel-03A7 opened the pull request:
## Description
<!-- Describe the motivation behind this PR -->
Fix sidebar icons alingment.

## Evidence
<!-- Attach pictures to show changes if applicable -->
Before
![image](https://github.com/Usiel-03A7/eventoseve/assets/58275665/2a5cbd38-6957-47f5-b13c-b1142898439e)
After
![image](https://github.com/Usiel-03A7/eventoseve/assets/58275665/0cf3b20a-3937-4271-98a0-6cee588b394e)



## Link Issue (if applicable)
  -

Usiel-03A7 committed:
Fix sidebar icons alignment

AlanAnguiano commented on src/components/sidebar/OptionsPanel/OptionItem/index.jsx:
```diff
@@ -2,26 +2,26 @@ import { useEffect, useState } from "react";
 
 export const OptionItem = ({ optionText, path, children }) => {
-  const [isActive, setActive] = useState(false)
+  const [isActive, setActive] = useState(false);
```
Avoid using ;

Usiel-03A7 replied:
done.

AlanAnguiano commented on src/components/sidebar/OptionsPanel/OptionItem/index.jsx:
```diff
@@ -2,26 +2,26 @@ import { useEffect, useState } from "react";
-  
-  useEffect(()=>{
-    const currentPath = window.location.pathname
+    return isActive ? styles.optionPanel__itemActive : "";
```
Same as above

AlanAnguiano commented on src/components/sidebar/OptionsPanel/OptionItem/index.jsx:
```diff
@@ -2,26 +2,26 @@ import { useEffect, useState } from "react";
+    const currentPath = window.location.pathname;
+
+    if (currentPath === path) {
+      setActive(true);
```
Same

AlanAnguiano commented on src/components/sidebar/OptionsPanel/OptionItem/index.jsx:
```diff
@@ -2,26 +2,26 @@ import { useEffect, useState } from "react";
-  )
-}
\ No newline at end of file
+  );
+};
```
Same, and also in all the lines that you find

Usiel-03A7 committed:
Add propTypes and esLint to package

Usiel-03A7 committed:
Add rules to file .esLintrc

Usiel-03A7 committed:
Modify optionItem

Modifies the component OptionItem based on EsLint conventions
and adds the required propTypes

AlejandroCuriel reviewed (approved):
LGTM BRO! 💯

AlanAnguiano reviewed (approved):
Shipt it :shipit:
